demos/README.md

```diff
@@ -24,6 +24,7 @@ This set of demos shows how real-world apps can be easily run inside SGX enclave
 * [sqlite](sqlite/) A demo of [SQLite](https://www.sqlite.org) SQL database engine.
 * [tensorflow_lite](tensorflow_lite/): A demo and benchmark of [Tensorflow Lite](https://www.tensorflow.org/lite) inference engine.
 * [xgboost](xgboost/): A demo of [XGBoost](https://xgboost.readthedocs.io/en/latest).
+* [pytorch](pytorch/): A demo of [Pytorch](https://pytorch.org/).
```

Comment:
Please put the pytorch demo before redis, the demos are sorted alphabetically.

Reply:
Done